app/pods/cases/case/subcases/overview/route.js

```diff
@@ -23,13 +18,10 @@ export default Route.extend({
       subcases,
       case: caze,
     });
-  },
-
-  actions: {
-    refresh() {
-      this._super(...arguments);
-      this.refresh();
-    },
-  },
+  }
```

Comment:
Zoals beschreven in de story kan de model hook beter gerefactored worden zodat het main model de lijst van subcases is en de case nadien ingesteld wordt als `@tracked` property op de controller.